python/lsst/pipe/drivers/multiBandDriver.py

```diff
@@ -68,6 +70,10 @@ def getPatchRefList(tract):
 
 class MultiBandDriverConfig(Config):
     coaddName = Field(dtype=str, default="deep", doc="Name of coadd")
+    doDetection = Field(dtype=bool, default=False, doc="Run detection on the "
+                        "Coadded exposure")
```

Comment:
The approach the rest of this script takes is to create the product if it doesn't exist. I think detection should follow that example.